src/components/ProcessDocumentScreen/ProcessedDocumentTag/ProcessedDocumentTag.tsx

```diff
@@ -45,35 +45,35 @@ export const ProcessedDocumentTag: FunctionComponent<PublishedTagProps> = ({
             className="mr-4 flex-shrink-0"
             data-testid="processing-loader"
             width="48px"
-            primary="#3B8CC5"
+            primary="#2D5FAA"
           />
```

Comment:
`primary` perhaps to put in `constants` folder and export from there

Reply:
ok